.editorconfig

```diff
@@ -11,6 +11,7 @@ insert_final_newline = true
 [*.{kt,kts,gradle}]
 indent_size = 4
 max_line_length = 120
+disabled_rules=import-ordering
```

Comment:
Don't disable lint checks. You can even autofix the import ordering (and almost all other issues) by running `./gradlew ktlintFormat`.